service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/CircuitBreakingServiceLocator.java

```diff
@@ -26,10 +23,22 @@
  */
 public abstract class CircuitBreakingServiceLocator implements ServiceLocator {
 
-    private final CircuitBreakers circuitBreakers;
+    private final CircuitBreakersPanel circuitBreakersPanel;
 
-    public CircuitBreakingServiceLocator(CircuitBreakers circuitBreakers) {
-        this.circuitBreakers = circuitBreakers;
+    public CircuitBreakingServiceLocator(CircuitBreakersPanel circuitBreakersPanel) {
+        this.circuitBreakersPanel = circuitBreakersPanel;
+    }
+
+
+
+    /**
+     * @deprecated Use constructor accepting {@link CircuitBreakersPanel} instead
+     * @param circuitBreakers
+     */
+    @Deprecated
+    public CircuitBreakingServiceLocator(com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers) {
```

Comment:
I think the full qualification of the class name here is leftover from an intermediate state of the PR when there were multiple types called `CircuitBreakers`.